eng/performance/scenarios.proj

```diff
@@ -5,41 +5,84 @@
       <Timeout>4:00</Timeout>
     </HelixWorkItem>
   </ItemDefinitionGroup>
   
+
   <ItemGroup>
-    <HelixWorkItem Include="Static Console Template">
+    <Scenario Include="Static Console Template">
       <PayloadDirectory>$(ScenariosDir)staticconsoletemplate</PayloadDirectory>
-    </HelixWorkItem>
-    <HelixWorkItem Include="Static VB Console Template">
+    </Scenario>
+    <Scenario Include="Static VB Console Template">
       <PayloadDirectory>$(ScenariosDir)staticvbconsoletemplate</PayloadDirectory>
-    </HelixWorkItem>
-    <HelixWorkItem Include="Static Winforms Template">
+    </Scenario>
+    <Scenario Include="Static Winforms Template">
       <PayloadDirectory>$(ScenariosDir)staticwinformstemplate</PayloadDirectory>
-    </HelixWorkItem>
-    <HelixWorkItem Include="New Console Template">
+    </Scenario>
+    <Scenario Include="New Console Template">
       <PayloadDirectory>$(ScenariosDir)emptyconsoletemplate</PayloadDirectory>
-    </HelixWorkItem>
-    <HelixWorkItem Include="New VB Console Template">
+    </Scenario>
+    <Scenario Include="New VB Console Template">
       <PayloadDirectory>$(ScenariosDir)emptyvbconsoletemplate</PayloadDirectory>
+    </Scenario>
+  </ItemGroup>
+
+  <!-- Startup FDD publish -->
+  <ItemGroup>
+    <HelixWorkItem Include="@(Scenario -> 'Startup - %(Identity) - FDD Publish')">
+      <PreCommands>$(Python) pre.py publish -f $(_Framework) -c Release</PreCommands>
+      <Command>$(Python) test.py startup --scenario-name &quot;%(Identity)&quot;</Command>
     </HelixWorkItem>
   </ItemGroup>
-</Project>
\ No newline at end of file
+
+  <!-- SOD FDD publish -->
+  <ItemGroup>
+    <HelixWorkItem Include="@(Scenario -> 'SOD - %(Identity) - FDD Publish')">
+      <PreCommands>$(Python) pre.py publish -f $(_Framework) -c Release</PreCommands>
+      <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
+    </HelixWorkItem>
+  </ItemGroup>
+
+  <!-- SOD SCD publish w/ RID -->
+  <ItemGroup>
+    <HelixWorkItem Include="@(Scenario -> 'SOD - %(Identity) - SCD Publish')">
+      <PreCommands>$(Python) pre.py publish -f $(_Framework) -c Release -r $(RID)</PreCommands>
+      <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)quot;</Command>
+    </HelixWorkItem>
+  </ItemGroup>
+
+  <!-- SOD build -->
+  <ItemGroup>
+    <HelixWorkItem Include="@(Scenario -> 'SOD - %(Identity) - Build')">
+      <PreCommands>$(Python) pre.py build -c Release -f $(_Framework)</PreCommands>
+      <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
+    </HelixWorkItem>
+  </ItemGroup>
+  <!-- This is useful for testing -->
+  <!-- <Target Name="t">
+        <Message Text="@(HelixWorkItem -> 'name: %(HelixWorkItem.Identity)
+     dir: %(HelixWorkItem.PayloadDirectory)
+     pre: %(HelixWorkItem.PreCommands)
+     command: %(HelixWorkItem.Command)
+     post: %(HelixWorkItem.PostCommands)
+     timeout: %(HelixWorkItem.Timeout)  '"/>
+  </Target> -->
```

Comment:
Should we just uncomment this?

Reply:
It turns out that even if you try to use `/t:test` (assuming it had a better name) including the Helix SDK makes a bunch of other stuff run anyway, so you have to manipulate the file before you can do local testing.

Reply:
Could we have that documented somewhere? I think just having this puts people who want to try and run it in a bad position. This could be in another PR.

Reply:
Done.